mailbox/lucene/src/main/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndex.java

```diff
@@ -430,23 +438,23 @@ protected Analyzer createAnalyzer(boolean lenient) {
             return new StrictImapSearchAnalyzer();
         }
     }
-    
+
     /**
      * If set to true this implementation will use {@link WildcardQuery} to match suffix and prefix. This is what RFC3501 expects but is often not what the user does.
      * It also slow things a lot if you have complex queries which use many "TEXT" arguments. If you want the implementation to behave strict like RFC3501 says, you should
-     * set this to true. 
-     * 
+     * set this to true.
+     *
      * The default is false for performance reasons
-     * 
-     * 
+     *
+     *
      * @param suffixMatch
      */
     public void setEnableSuffixMatch(boolean suffixMatch) {
         this.suffixMatch = suffixMatch;
     }
-    
-    
-    
+

```

Comment:
Really, reformat in commits without code changes, please....

It would make review 100x easier